### DISCUSSION TITLE
Using pkc alongside other plex addons

### DISCUSSION THREAD

**Discussion by djnield** [croneter/PlexKodiConnect#1432] [Q&A] [answered]
Could you expand a little why we shouldn't have other plex addons installed when using pkc?

I also have the official kodi plex app on my system.

**Comment by croneter** [comment, marked as answer]
It's possible that certain "variables" are used by both PlexKodiConnect as well as another Plex add-on. I have also seen that the official Plex for Kodi add-on intervenes with PlexKodiConnect's playback reports. 

Try both add-ons simultaneously - you'll notice if it won't work 😉 .
